### ISSUE TITLE
Print dialog with greyed out options

### ISSUE THREAD

**Issue by eric0001** [elementary/evince#10]
In elementary OS 6 Beta 2 (Odin) the print dialog of evince pdf reader (default elementary flatpak version) has some options greyed out and also is missing a tab. These options and this tab are present in the print dialog of evince pdf reader (the deb repository version).
![evince-flatpak-print-dialog-bug](https://user-images.githubusercontent.com/39679934/126074709-e7103e00-8afe-43a5-8c53-1acdafc29e2b.jpg)
![evince-deb-print-dialog-bug](https://user-images.githubusercontent.com/39679934/126074715-15359384-3488-437b-8cd0-c478c2765488.jpg)

**Comment by queeup** [comment]
They didn't activate printing access on the evince flatpak app. You need to enable it yourself.
`System Settings > Applications > Permissions > Evince > Printing`

**Comment by eric0001** [comment]
Of course I did enable myself the printing access in the System Settings. And I am able to print. The issue is not about being able to print in Evince. The issue here is about options that are available in the apt (.deb) version of Evince but are not available in the default elementary flatpak version of Evince. I suspect an issue with the way the flatpak of Evince has been built in elementary's appcenter repo.
I also noticed right now that the preview button of the print dialog (in elementary's default Evince) does not work either. But the preview button does work in the apt .deb version of Evince.
